content/articles/why-we-dont-do-webhosting.markdown

```diff
@@ -1,19 +1,18 @@
 ---
 title: Why we don't offer web hosting services
-excerpt: Welcome to DNSimple. This support page explains why we don't offer web hosting along our DNS hosting.
+excerpt: Some of the reasons why DNSimple does not offer web hosting services.
 categories:
 - DNSimple
 ---
 
 # Why we don't offer web hosting services
 
-Traditionally DNS registrars have been offering domains bundled with a hosting solution as part of a basic package. This usually translated into giving a few MegaBytes of disk space exposed by an HTTP server and accessible with a FTP client for the user. The idea behind was that the user could have her personal HTML static site. 
+Traditionally, DNS registrars have offered domains bundled with a hosting solution as part of a basic package. This usually translates into giving a few MegaBytes of disk space exposed by an HTTP server and accessible with a FTP client for the user. The idea behind this was that the user could have their personal HTML static site bundled in with the registrar's offerings.
 
 This can cause confusion to people without deep understanding of these technologies. This article explains why DNSimple does not offer web hosting. Some of the reasons are:
 
- * We do one thing and We do it very well, that is offering the best DNS hosting experience. 
- * Today the web is way more diverse and complex. There is a myriad of technologies available for creating websites. These technologies use several pieces of software to work, database servers, web servers, caching services, load balancers, etc. Websites have heavier hardware requirements than some Megabytes of hard disk. 
+ * We do one thing and We do it very well, that is offering the best DNS hosting experience.

```

Comment:
We can maybe be a bit more specific. Perhaps something like:

We prefer to focus our efforts on offering the best ever domain management platform by enabling our customers to set up and connect their domains to popular services that already exist, rather than trying to create our own top-to-bottom solution.